homework/content.js

```diff
@@ -1,37 +1,39 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
-
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+(function(exports) {                                 //exports是自己命名的 其實就是window
+  var ListContentManager = function(){
+    this._wrapper = null
+  });
+ 
+  ListNoteContentManager.prototype = {
+      start() {
+              this._wrapper = document.querySelector('#note-content-wrapper');
+              window.addEventListener('note-open', (function(event) {
```

Comment:
Instead of binding an anonymous function, you should bind the this as the listener. And Arrow function is better than the binding API (it's more concise), although I didn't require you to use that. See:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Lexical_this